_includes/cloudcode/cloud-code-advanced.md

```diff
@@ -583,6 +583,18 @@ Here's an example of the JSON data that would be sent in the request to this web
 
 After setting up your webhook in the Dashboard UI, you'll be acurately decrementing comment counts!
 
+# Sending Emails
+If your Parse Server configuration specifies an `emailAdapter`, you can send mail via cloud code using `Parse.Cloud.sendMail`.
```

Comment:
I just had another look at the Parse Server options. The option is called `emailAdapter`, only internally the adapter is called `MailAdapter` (without an `e`), also the adapter's method is called `sendMail`. Throughout the docs, there is no reference to `mail`, only `email`.

I think this should be harmonized internally and externally, and we should add an `e` also internally at some point, but at least it should be uniform externally. So I suggest to change this so:

> If an `emailAdapter` is configured in the [Parse Server Options](https://parseplatform.org/parse-server/api/4.5.0/ParseServerOptions.html), you can send an email via Cloud Code using `Parse.Cloud.sendEmail(...)`.